conf/machine/include/rpi.inc

```diff
@@ -18,9 +18,6 @@ CMDLINE_ROOTFS = ""
 # u-boot config vars
 UBOOT_FDT_ADDR = "fdt_addr"
 
-# ethernet device name
-OMNECT_ETH0 = "eth0"
-
 # wlan device name
 OMNECT_WLAN0 = "wlan0"
```

Comment:
when we remove `OMNECT_ETHX`, why don't we remove `OMNECT_WLAN0` as well?. as far as i understand, systemd will name it `wlan0` anyways.